src/main/resources/templates/db_scripts/schema.json

```diff
@@ -373,7 +373,7 @@
       "uniqueIndex": [
         {
           "fieldName": "encumbrance",
-          "multiFieldNames": "amount, fromFundId, encumbrance.sourcePurchaseOrderId, encumbrance.sourcePoLineId, encumbrance.initialAmountEncumbered, encumbrance.status, expenseClassId, fiscalYearId",
+          "multiFieldNames": "amount, fromFundId, encumbrance.sourcePurchaseOrderId, encumbrance.sourcePoLineId, encumbrance.initialAmountEncumbered, expenseClassId, fiscalYearId",
```

Comment:
with this change we touching business logic, it's unique index, we should not change it declaration